### DISCUSSION TITLE
2FA Authenticaton in R Shiny

### DISCUSSION THREAD

**Discussion by yohanespm77** [teamalgoritma/community#877] [Q&A] [answered]
Adakah referensi untuk membuat login page dengan 2FA Authentication di R Shiny ?

Mohon bantuannya kak, terima kasih

**Comment by ramdhinafinita** [comment, marked as answer]
Dear @yohanespm77 

Untuk penggunaan 2FA auth sepertinya belum ada secara spesifik bisa dilakukan. Namun, anda bisa coba merefer pada link berikut:
- https://ploomber.io/blog/shiny-r-auth/
- https://github.com/treysp/shiny_password
- https://www.listendata.com/2019/06/how-to-add-login-page-in-shiny-r.html

**Comment by ramdhinafinita** [comment]
Halo @yohanespm77, untuk menjaga agar forum diskusi berjalan kondusif, karena tidak ada respon lanjutan setelah 7 hari dari respon terakhir, kami izin menandai pertanyaan sudah terjawab dan menutup diskusi ini. Semoga dapat dipahami dan terima kasih.
